configurations/manager/large_dataset.yaml

```diff
@@ -0,0 +1,5 @@
+test_duration: 1200
+
+instance_type_db: 'i3.xlarge'
+
+stress_cmd: "cassandra-stress write cl=QUORUM n=200200300 -schema 'replication(strategy=NetworkTopologyStrategy,us-eastscylla_node_east=2,us-west-2scylla_node_west=1)' -port jmx=6868 -mode cql3 native -rate threads=500 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=400200300..600400600"
```

Comment:
that sounds like about 200GB total data set. perhaps it be useful to document it somewhere in the code. you may wish to do some comparisons/tuning/measurements later on.

Reply:
Actually, it's (nearly) exactly 200GB (I looked at the output of the manager backup). Where should I document it?

Reply:
> Actually, it's (nearly) exactly 200GB (I looked at the output of the manager backup). Where should I document it?

IMHO in the file name

Reply:
Done and done